packages/core/src/Agentica.ts

```diff
@@ -140,13 +158,16 @@ export class Agentica {
       text: content,
     };
     await this.dispatch(prompt);
+    const context = this.getContext({ prompt, usage: this.token_usage_ });
 
-    const newbie: IAgenticaPrompt[] = await this.executor_(
-      this.getContext({
-        prompt,
-        usage: this.token_usage_,
-      }),
-    );
+    const middlewares = this.middlewares_.get("conversate" as const);
```

Comment:
I recommend this `"conversate"` string literal convert string literal constant variable